qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -138,13 +191,7 @@ def _choose_layout_condition(property_set):
     else:
         raise TranspilerError("Invalid layout method %s." % layout_method)
 
-    def _not_perfect_yet(property_set):
-        return (
-            property_set["trivial_layout_score"] is not None
-            and property_set["trivial_layout_score"] != 0
-        )
-
-    # 4. Extend dag/layout with ancillas using the full coupling map
+    # 3. Extend dag/layout with ancillas using the full coupling map
```

Comment:
```suggestion
    # 4. Extend dag/layout with ancillas using the full coupling map
```

(doesn't look like the number of steps has changed - there's still a 3 above this)